src/stream.c

```diff
@@ -644,6 +675,11 @@ void LIBUSB_CALL _uvc_stream_callback(struct libusb_transfer *transfer) {
   
   if ( strmh->running && resubmit )
     libusb_submit_transfer(transfer);
+	
+  if ( strmh->running && resubmit )
+    libusb_submit_transfer(transfer);
+  else
+    _uvc_delete_transfer(transfer);
```

Comment:
This function seems to be a copy of the code in the LIBUSB_TRANSFER_CANCELLED / LIBUSB_TRANSFER_ERROR / LIBUSB_TRANSFER_NO_DEVICE case of the switch statement above.  That code above can free the transfer, which means the first line of _uvc_delete_transfer, `uvc_stream_handle_t *strmh = transfer->user_data;`, will be accessing free'd memory.